massgov/openmass
# DP-22534: Make paragraph for News on org page

**Description:**
Adding an Organization News paragraph, making it addable in the Organization Sections paragraph, and refactoring related logic to apply to the new paragraph.

**Jira:** (Skip unless you are MA staff)
https://massgov.atlassian.net/browse/DP-22534


**To Test:**
- Login and go to /node/add/org_page (or edit an existing org_page).
- Click on the "Content" tab.
- Expand the "Section Content" field.
- [ ] Verify there is an "Add Organization News" button.
- Click the "Add Organization News" button.
- [ ] Verify these fields are present: Featured News items, Automatic news items quantity, Show images for automatic news items.
- Add content to the fields and save (recommend editing an existing org_page with News content).
- [ ] View the org_page and verify the content displays as expected.

**Screenshots/GIFs:**
![screencapture-mass-local-orgs-massachusetts-registry-of-motor-vehicles-2021-07-16-17_11_45](https://user-images.githubusercontent.com/1434979/126009008-e954cf20-6588-40f4-b1b9-9d59ad70fe01.png)

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
